src/components/Table/Pagination.react.js

```diff
@@ -57,7 +57,7 @@ export class Pagination extends React.Component {
 
     pageLinks.push(
       <li key="last">
-        <a onClick={this.goToLastPage.bind(this)} style={ {cursor: "pointer"} }>{ t('pagination.last') }</a>
+        <a onClick={this.goToLastPage.bind(this)} style={ {cursor: "pointer"} }>{ t('filterbar.pagination.last') }</a>
```

Comment:
can you please recheck the format with this PR https://github.com/rdytech/neptune/pull/8727/files#diff-a5a86160b667645873b275b84f4524f4ade7347bbbd25813378cb148d0826be3R1350-R1352

Reply:
ah all good I saw it updated in your PR https://github.com/rdytech/neptune/pull/8799